src/Components/Forms/OrderForm/App.tsx

```diff
@@ -60,7 +60,7 @@ const StepContainer = styled.div`
   margin: 0 auto;
 `
 
-export const StyledTitle = Title.extend`
+export const StyledTitle = styled(Title)`
```

Comment:
Each of these `styled(...)` replacements is due to `styled-components` removing this api method in version 4 with the warning failing our tests.